projects/media-viewer/src/lib/annotations/annotation-set/annotation/comment/comment.component.ts

```diff
@@ -12,20 +12,21 @@ export class CommentComponent {
 
   private readonly MAX_COMMENT_LENGTH = 50;
 
-  editable = false;
   lastUpdate: string;
   fullComment: string;
   author: User;
   editor: User;
+  _editable: boolean;
```

Comment:
eh? we've never used _ notation before... any reason to start now?

Reply:
it's because I had to introduce the setters/getters for the reasons explained below